build-system/tasks/extension-helpers.js

```diff
@@ -441,13 +441,16 @@ function buildExtension(
   if (options.watch) {
     options.watch = false;
     watch(path + '/**/*', function() {
-      buildExtension(
+      const complete = buildExtension(
```

Comment:
Maybe use the same variable name for `complete` (here) and `bundlePromise` (in `lazy-build.js`) for future maintainability?